modules/influxdb/src/main/java/org/testcontainers/containers/InfluxDBContainerV1.java

```diff
@@ -0,0 +1,165 @@
+package org.testcontainers.containers;
+
+import java.util.Collections;
+import java.util.Set;
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * See <a href="https://store.docker.com/images/influxdb">https://store.docker.com/images/influxdb</a>
+ */
+public class InfluxDBContainerV1<SELF extends InfluxDBContainerV1<SELF>> extends GenericContainer<SELF> {
+
+    public static final Integer INFLUXDB_PORT = 8086;
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("influxdb");
+    private static final String DEFAULT_TAG = "1.4.3";
+
+    private static final int NO_CONTENT_STATUS_CODE = 204;
+
+    @Deprecated
+    public static final String VERSION = DEFAULT_TAG;
+
+    private boolean authEnabled = true;
+    private String admin = "admin";
+    private String adminPassword = "password";
+
+    private String database;
+    private String username = "any";
+    private String password = "any";
+
+    /**
+     * @deprecated use {@link InfluxDBContainer(DockerImageName)} instead
+     */
+    @Deprecated
+    public InfluxDBContainerV1() {
+        this(DEFAULT_IMAGE_NAME.withTag(DEFAULT_TAG));
+    }
+
+    /**
+     * @deprecated use {@link InfluxDBContainer(DockerImageName)} instead
+     */
+    @Deprecated
+    public InfluxDBContainerV1(final String version) {
+        this(DEFAULT_IMAGE_NAME.withTag(version));
+    }
+
+    public InfluxDBContainerV1(final DockerImageName dockerImageName) {
+        super(dockerImageName);
+
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+
+        this.waitStrategy = new WaitAllStrategy()
+            .withStrategy(Wait.forHttp("/ping").withBasicCredentials(this.username, this.password)
+                .forStatusCode(NO_CONTENT_STATUS_CODE))
+            .withStrategy(Wait.forListeningPort());
+
+        this.addExposedPort(INFLUXDB_PORT);
+    }
+
+    @Override
+    protected void configure() {
+        this.addEnv("INFLUXDB_ADMIN_USER", this.admin);
+        this.addEnv("INFLUXDB_ADMIN_PASSWORD", this.adminPassword);
+
+        this.addEnv("INFLUXDB_HTTP_AUTH_ENABLED", String.valueOf(this.authEnabled));
+
+        this.addEnv("INFLUXDB_DB", this.database);
+        this.addEnv("INFLUXDB_USER", this.username);
+        this.addEnv("INFLUXDB_USER_PASSWORD", this.password);
+    }
+
+    @Override
+    public Set<Integer> getLivenessCheckPortNumbers() {
+        return Collections.singleton(this.getMappedPort(INFLUXDB_PORT));
+    }
```

Comment:
Minor nit, and not at all anything you've done. I think this method is redundant, as the listening port wait strategy.

Let's remove this in a future PR, though!